core/src/main/java/com/sequenceiq/cloudbreak/service/upgrade/validation/service/NifiUpgradeValidator.java

```diff
@@ -0,0 +1,67 @@
+package com.sequenceiq.cloudbreak.service.upgrade.validation.service;
+
+import java.util.Optional;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.cluster.api.ClusterApi;
+import com.sequenceiq.cloudbreak.cmtemplate.CmTemplateService;
+import com.sequenceiq.cloudbreak.common.exception.UpgradeValidationFailedException;
+import com.sequenceiq.cloudbreak.domain.stack.Stack;
+import com.sequenceiq.cloudbreak.service.cluster.ClusterApiConnectors;
+
+@Component
+public class NifiUpgradeValidator implements ServiceUpgradeValidator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NifiUpgradeValidator.class);
+
+    private static final String NIFI_SERVICE_TYPE = "NIFI";
+
+    private static final String NIFI_WORKING_DIRECTORY = "nifi.working.directory";
```

Comment:
Is this enough to check for?

I can see the following setting, too, it can be set to root, too:

```class ApiConfig {
    name: nifi.conf.directory
    value: null
    required: false
    _default: ${CONF_DIR}
    displayName: NiFi Configuration Directory
    description: Directory with all configurations for NiFi daemons
    relatedName: nifi.conf.directory
    sensitive: false
    validationState: OK
    validationMessage: null
    validationWarningsSuppressed: false
}
```
cc: @keyki

Reply:
As I know it's enough to check the working directory. Maybe @vbrodetskyi can answer this.

Reply:
@pdarvasi according to what i know, OS upgrade should be available/visible ONLY if NiFI/NiFI Registry working directory is located on attached volumes, because in that case we will be able to save required data. @pvillard31 could you please approve my asnwer or correct me if i'm wrong?

Reply:
Correct - that's also my expectation/understanding.

Reply:
thanks a lot, so just to be 100% sure: that means that
`nifi.conf.directory`
**need not** be checked, right?